.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```

Comment:
Could you please add a new line at the end of the file? Just a tiny best practice thing.

Reply:
added a new line at the end of the file.